app/models/user.server.ts

```diff
@@ -52,7 +52,8 @@ export async function createUser(
   username: PrismaUser['username'],
   firstName: PrismaUser['firstName'],
```

Comment:
Drop `firstName` and `lastName` and use `displayName` or just `name` if you want (both parts are given to us in 1 single name).

Reply:
Got it!